numpy/_version.py

```diff
@@ -498,7 +498,7 @@ def get_versions():
         # versionfile_source is the relative path from the top of the source
         # tree (where the .git directory might live) to this file. Invert
         # this to find the root from __file__.
-        for i in cfg.versionfile_source.split('/'):
+        for root in cfg.versionfile_source.split('/'):
```

Comment:
This is wrong. Please don't change this line.

Reply:
This is generated code, we should not be changing it at all

Reply:
I made #19819 to make be marked as generated in future PRs.